src/org/labkey/test/pages/study/DatasetDesignerPage.java

```diff
@@ -272,5 +300,19 @@ protected Locator dataRowRadioBtn(Integer index)
 
         protected final Checkbox keyPropertyManagedBox = new Checkbox(Locator.inputById("keyPropertyManaged")
                 .findWhenNeeded(propertiesPanel));
+
+        // this is only shown when inferring fields from a file
+        protected ToggleButton autoImportSlider()
```

Comment:
Use consistent naming.
```suggestion
        protected ToggleButton autoImportToggle()
```